pkgs/servers/jigasi/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
+
+let
+  pname = "jigasi";
+  version = "1.1-195-g65ef768";
+  arch = "amd64";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_${arch}.deb";
+    sha256 = "0fq4gb4pib5bm5bdh9qr1xszzwn8ryar0n3a32s333zq5hpv9xb4";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  dontBuild = true;
+
+  unpackCmd = "${dpkg}/bin/dpkg-deb -x $src debcontents";
```

```suggestion
  unpackCmd = "dpkg-deb -x $src debcontents";
```
Please use nativeBuildInputs instead of hard coding the path.